app/src/main/java/edu/uwstout/p2pchat/SettingFragment.java

```diff
@@ -55,6 +55,7 @@ public View onCreateView(final LayoutInflater inflater, final ViewGroup containe
         //Save context for shared preferences.
         context = view.getContext();
 
+        this.getActivity().getApplication();
```

Comment:
What was the purpose of this line of code? You get the application but you don't store it or use it?